### ISSUE TITLE
`rosa init` should report all quota errors it encounters so a single service request can be opened.

### ISSUE THREAD

**Issue by lijcam** [openshift/rosa#380]
I'm testing rosa on a limited aws subscription, when `rosa init` encounters a quota limitation it stop and notifies the user. But after increasing the quota to rosa's expectation, it reports the next quota limitation.

It would be great it  `rosa init` could report all the limitations at once so the user only needs to open a single service request.

```
I: Validating AWS credentials...
I: AWS credentials are valid!
I: Validating SCP policies...
I: AWS SCP policies ok
I: Validating AWS quota...
E: Insufficient AWS quotas
E: Service ebs quota code L-FD252861 Provisioned IOPS SSD (io1) volume storage not valid, expected quota of at least 300, but got 50
```

**Comment by jharrington22** [comment]
Hi @lijcam thanks for the feedback. It would be more helpful to the user if we reported them all at once I agree, we can take a look at this.

**Comment by doughadfield** [comment]
I have the same error (quota code L-FD252861) but this code does not seem to exist in AWS - the code for "IOPS for Provisioned IOPS SSD (io1)" shows as "L-B3A130E6" in AWS - am I looking in the wrong place, or is this a bug in rosa?

**Comment by vkareh** [comment]
Fixed by #386

**Comment by vkareh** [comment]
My bad - this issue is not fixed, still need to report all quota errors at once, rather than one at a time. Repoening.

**Comment by jharrington22** [comment]
Fixed by #390 :)

**Comment by jharrington22** [comment]
@doughadfield the quota code should exist for you try:

```
aws service-quotas get-service-quota --service-code ebs --quota-code "L-FD252861" --region us-east-1            
{
    "Quota": {
        "ServiceCode": "ebs",
        "ServiceName": "Amazon Elastic Block Store (Amazon EBS)",
        "QuotaArn": "arn:aws:servicequotas:us-east-1:xxxxxxxxx:ebs/L-FD252861",
        "QuotaCode": "L-FD252861",
        "QuotaName": "Storage for Provisioned IOPS SSD (io1) volumes, in TiB",
        "Value": 50.0,
        "Unit": "None",
        "Adjustable": true,
        "GlobalQuota": false
    }
}

```
